<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget 
{
	border-image: url(:/resource/map/map.png);
}
#Widget * 
{
	border-image:url();
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>248</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>长春</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>193</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>哈尔滨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>448</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>成都</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>318</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>银川</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>288</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>北京</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>318</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>天津</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>448</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>重庆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>乌鲁木齐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>356</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>西宁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>西安</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>500</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>贵阳</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>沈阳</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>378</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>兰州</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>拉萨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>528</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>昆明</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>呼和浩特</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>太原</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>石家庄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>济南</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>郑州</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>合肥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>南京</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>上海</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>武汉</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>长沙</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>480</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>南昌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>杭州</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>福州</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_28">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>510</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>台北</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>550</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>南宁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>海口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_31">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>540</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>广州</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>560</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>香港</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>澳门</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>301</width>
     <height>512</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>当前时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_8">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>当前城市:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_9">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>目的城市:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_10">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>下一目的城市:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_17">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>交通工具:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_11">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>旅客风险:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_12">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>出行策略:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_13">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_37">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>更改</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>出发时间:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_40">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_38">
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>-1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_14">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_39">
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>+1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_41">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>+10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>时间限制:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_42">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_43">
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>-1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_15">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_44">
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>+1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_45">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>+10</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>当前路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_16">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="pushButton_46">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_47">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
